demo/server/src/utils/context.ts

```diff
@@ -1,22 +1,49 @@
+import { PointsWriter, PointsReader } from '@ceramic-solutions/points'
 import { getAuthenticatedDID } from '@ceramic-solutions/did-utils'
 import { fromString } from 'uint8arrays'
-import { CeramicClient } from '@ceramicnetwork/http-client'
 import 'dotenv/config.js'
 
 type Context = {
-  ceramic: CeramicClient
-  aggregationModelID: string
+  contextWriter: PointsWriter
+  totalWriter: PointsWriter
+  contextReader: PointsReader
+  totalReader: PointsReader
 }
 
 export const getContext = async (): Promise<Context> => {
-  const CERAMIC_URL: string = process.env.CERAMIC_URL || ''
+  const CERAMIC_URL: string | undefined = process.env.CERAMIC_URL || undefined
   const CERAMIC_PRIVATE_KEY: string = process.env.CERAMIC_PRIVATE_KEY || ''
-  const aggregationModelID: string = process.env.AGGREGATION_ID || ''
+  const aggregationModelID: string | undefined = process.env.AGGREGATION_ID || undefined
 
```

Comment:
i would like a ceramic context that is independent of the model since i'll use it with both aggregation and allocation models.